examples/vae/train.py

```diff
@@ -116,14 +49,10 @@ def compute_metrics(recon_x, x, mean, logvar):
   }
 
 
-def model():
-  return VAE(latents=FLAGS.latents)
-
-
-@jax.jit
```

Comment:
A question for understanding, should the train step be jitted? I looked at other examples and that didn't seem to be the case. 

https://github.com/google/flax/blob/main/examples/wmt/train.py#L166

Reply:
it's jitted here:
https://github.com/google/flax/blob/df66c8128ede2a6b5642750f5c7c6e6e72605f1f/examples/wmt/train.py#L528

(the `pmap()` transform also compiles the code like `jit()`, but at the same time parallelizes it onto multiple devices)

Reply:
> should the train step be jitted?

to answer you question: yes, you should always compile the largest possble code block; usually that's the `train_step()`.

Reply:
Ah I missed it that, was looking for the decorator. Thank you @andsteing